earth_enterprise/src/fusion/autoingest/AssetFactory.h

```diff
@@ -225,7 +226,7 @@ namespace AssetFactory
       using Impl = typename MutableDerivedVersionHandleType::Impl;
       auto ref_ = AssetDefs::SubAssetName(parentAssetRef,
                                           basename,
-                                          type_,
+                                          type_, //Ipl::EXPECTED_TYPE
```

Comment:
Probably ought to get rid of this comment. At a minimum, it should be `Impl`, not `Ipl`.